misc/services/camel/de-metas-camel-edi/Dockerfile

```diff
@@ -6,7 +6,7 @@ FROM openjdk:17-jdk-alpine
 # thx to https://github.com/moby/moby/issues/1996#issuecomment-185872769
 ARG CACHEBUST=1
 
-RUN apk update && apk upgrade && apk add bash && apk add curl && apk add dos2unix --update-cache --repository http://dl-3.alpinelinux.org/alpine/edge/community/ --allow-untrusted
+RUN apk update && apk upgrade && apk add bash && apk add curl --update-cache --repository http://dl-3.alpinelinux.org/alpine/edge/community/ --allow-untrusted
 
 COPY ./target/de-metas-camel-edi-exec.jar /opt/metasfresh-esb-camel/
```

Comment:
I don't think that's the way to go here, we had dos2unix in place to take care of the line separators conversion between windows and linux ( at least that's what I've understood), so I'd imagine we still need that...

also, another thing to consider is that there are other Dockerfiles where we do the exact same thing for other shell scripts... and there everything goes well, the only difference being the package manager (i.e. apt instead of apk) so most probably we end up with different versions of the dos2unix utility.

now, I've looked at https://dl-3.alpinelinux.org/alpine/edge/community, the `dos2unix` source in `de-metas-camel-edi/Dockerfile` and I've noticed the package was updated on 13th of April 2023 => most probably that's the root cause of the failure.

Reply:
but ofc, if it's fine to just drop it, then let's save some effort and just drop it :)

Reply:
I was told by @metas-ts that dos2unix is most likely no longer needed. 
When I attempted the docker build myself it ran just fine. Only jenkins produced the 139 Error every single time. Maybe its due to the nested and temporary directories and they no longer play fine with the latest version of the dos2unix tool? Just an assumption.